src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![deny(rustdoc::broken_intra_doc_links)]
+#![deny(rustdoc::private_intra_doc_links)]
+
+mod vss_client;
+mod vss_error;
```

Comment:
Why do we we use the "vss_" prefix for modules? Users of the crate already need to give the crate name, so this results in a redundant `vss_client`.